content/sensu-go/5.17/guides/email-handler.md

```diff
@@ -0,0 +1,311 @@
+- If your event status stays at `1` for the next hour, you **will not** receive repeated email notifications every 60 seconds during that hour
+- If your event status changes to `2` after 1 hour at `1`, you will receive **one** email notification for the change from `warning` status to `critical` status
+- If your event status fluctuates between `0`, `1`, and `2` for the next hour, you will receive **one** email notification **each time** the status changes
+
+Adding the event filter requires two parts:
+
+1. Create the event filter
+2. Create the email handler definition to specify the email address where the `sensu/sensu-email-handler` asset will send incident notifications
+
+First, to create the event filter, run:
+
+{{< highlight shell >}}
+cat << EOF | sensuctl create
+---
+type: EventFilter
+api_version: core/v2
+metadata:
+  annotations: null
+  labels: null
+  name: state_change_only
+  namespace: default
+spec:
+  action: allow
+  expressions:
+  - event.check.occurrences == 1
+  runtime_assets: []
+EOF
+{{< /highlight >}}
+
+Second, create the email handler definition.
+In the handler definition's `command` value, you'll need to change a few things.
+
+Copy this text into a text editor:
+
+{{< highlight shell >}}
+cat << EOF | sensuctl create
+---
+api_version: core/v2
+type: Handler
+metadata:
+  namespace: default
+  name: email
+spec:
+  type: pipe
+  command: sensu-email-handler -f YOUR-SENDER@example.com -t YOUR-RECIPIENT@example.com -s YOUR-SMTP-SERVER.example.com
+    -u USERNAME -p PASSWORD
+  timeout: 10
+  filters:
+  - is_incident
+  - not_silenced
+  - state_change_only
+EOF
+{{< /highlight >}}
+
+Then, replace the following text:
+
+- `YOUR-SENDER@example.com`: Replace with the email address you want to use to send email alerts
+- `YOUR-RECIPIENT@example.com`: Replace with the email address you want to receive email alerts
+- `YOUR-SMTP-SERVER.example.com`: Replace with the hostname of your SMTP server
+- `USERNAME`: Replace with your SMTP username, typically your email address
+- `PASSWORD`: Replace with your SMTP password, typically the same as your email password
+
+_**NOTE**: To use Gmail or G Suite as your SMTP server, follow Google's instructions for [sending email via SMTP][14]. If you have enabled 2-step verification on your Google account, you'll need to use an [app password][15] instead of your login password._
+
+You probably noticed that the handler definition includes two other filters: [`is_incident`][10] and [`not_silenced`][11].
+These two filters are included in every Sensu backend installation, so you don't have to create them.
+
+After you add your email, server, username, and password values, run your updated code to create the email handler definition.
+
+Now your handler and event filter are set up!
+Next, assign the email handler to your `server-health` check.
+
+## Assign the email handler to a check
```

Comment:
I don't think this step is necessary, as the handler is being specified in our ad-hoc event payloads.